src/dioptra/restapi/v1/queues/schema.py

```diff
@@ -19,37 +19,25 @@
 
 from marshmallow import Schema, fields
 
-from dioptra.restapi.v1.groups.schema import GroupRefSchema
 from dioptra.restapi.v1.schemas import (
     BasePageSchema,
     GroupIdQueryParametersSchema,
     PagingQueryParametersSchema,
     SearchQueryParametersSchema,
+    generate_base_resource_ref_schema,
     generate_base_resource_schema,
 )
 
+QueueRefBaseSchema = generate_base_resource_ref_schema("Queue")
+
 
 class QueueRefSchema(Schema):
```

Comment:
The class definition `QueueRefSchema(Schema):` should be `QueueRefSchema(QueueRefBaseSchema):` I believe.